corehq/apps/app_manager/tests/test_suite_multi_select_case_list.py

```diff
@@ -297,14 +298,16 @@ def setUp(self):
         self.m3, m3f0 = self.factory.new_basic_module('m3', self.OTHER_CASE_TYPE)
         m3f0.requires = 'case'
 
+        self._render_suite.reset_cache(self)
```

Comment:
:+1: thanks, I was planning on doing something to that effect, this is a clean approach